Entities/ACraft.cpp

```diff
@@ -246,6 +246,9 @@ MOSRotating * ACraft::Exit::SuckInMOs(ACraft *pExitOwner)
     return 0;
 }
 
+// Initialise static flag to check for active crab bombs.
+bool ACraft::m_sCrabBombInEffect = false;
```

Comment:
Rename this to `s_CrabBombInEffect`, put it on the top before `Clear()` and remove the comment.